src/main/webapp/default-site/scripts/rest/api/1/server/get-ui-resource-override.get.groovy

```diff
@@ -20,5 +20,7 @@ if(resource != null) {
 }
 else {
 	response.setStatus(404)
-	return "NOT FOUND"
+    def instance = applicationContext.get("instanceService")
```

Comment:
Is this related to the instance ID?

Reply:
I have removed that with the new commit. It was a mistake.